app/src/main/java/com/spielpark/steve/bernieapp/bernrate/BernRateFragment.java

```diff
@@ -85,12 +52,38 @@ private void displayWebViews(String[] urls) {
         }
     }
 
+    private String[] generateURLs() {
+        String[] gen = new String[urls.length];
+        int[] heightWidth = Util.getScreenWidthHeight(getActivity());
+        for (int i = 0; i < gen.length; i++) {
+            gen[i] = urls[i].concat(" width=\"" + heightWidth[0] + "px\" height=\"" + heightWidth[1] + "px\"></iframe>");
```

Comment:
While you're refactoring, this whole `iframe` part seems like it could be broken up

Feel like `generateURLs` should handle the tags as part of the string building, and the `urls` should really only be the `urls`.

Something like
```java
private static final String BERNRATE_BASE_URL = "http://bernrate.com/embed?datakey=";
private static final String[] keys = new String[](
  "followers:berniesandres",
  "tweet:feelthebern"
);
//..


sb.append("<iframe ");
sb.append("src=\"");
sb.append(BERNRATE_BASE_URL);
sb.append(keys[i]);
sb.append("\"  width=\"");
sb.append(heightWidth[0]);
//...
```

(string builder just as an example to illustrate `urls` only containing the actual urls, and